src/Rector/New_/StringToArrayArgumentProcessRector.php

```diff
@@ -34,9 +29,13 @@
  */
 final class StringToArrayArgumentProcessRector extends AbstractRector
 {
+    /**
+     * @var string[]
+     */
+    private const ALLOWED_PROCESS_METHOD_CALLS = ['setWorkingDirectory'];
```

Comment:
```suggestion
    private const EXCLUDED_PROCESS_METHOD_CALLS = ['setWorkingDirectory'];
```

Reply:
I'm not sure why there are `MethodCall` in the first place.
Maybe it would be better to `INCLUDED_PROCESS_METHOD_CALLS` explicitly and name the methods, that can be used in this rule.

Reply:
updated

Reply:
I think this may be the wrong approach. Pretty much any method in the Process that uses a string variable is being converted to array arguments. It seems that only the constructor should be changed from a string to array. Not every method call in the class. Maybe I'm missing something. Take a look at the other public methods `addOutput` and `addErrorOutput` they have the same issue as my original #67